doc/en/pom.xml

```diff
@@ -24,234 +24,426 @@
       <plugin>
         <groupId>io.swagger</groupId>
         <artifactId>swagger-codegen-maven-plugin</artifactId>
-        <version>2.2.2</version>
+        <version>2.4.12</version>
+        <configuration>
+          <language>html</language>
+          <ignoreFileOverride>${project.basedir}/api/.swagger-codegen-ignore</ignoreFileOverride>
+        </configuration>
         <executions>
           <!-- one execution per page -->
           <execution>
             <id>coverages</id>
             <phase>process-resources</phase>
-            <goals>
-              <goal>generate</goal>
-            </goals>
+            <goals><goal>generate</goal></goals>
```

Comment:
I think I liked the original version better. Not a big deal though.

Reply:
I had a lot of cut and paste to get through :)